### DISCUSSION TITLE
How to capture multiple images from camera

### DISCUSSION THREAD

**Discussion by gkasireddy202** [NitrogenZLab/react-native-multiple-image-picker#185] [Q&A]
Does this npm have the functionality to capture multiple images from the camera?

**Comment by baronha** [comment]
Not currently but will be in the upcoming version but I think it only serves for single shooting. If possible, I will allow multiple photos.
Thank you for your interest

**Comment by gkasireddy202** [comment]
@baronha - Thanks for your update.
I captured a single image from the camera in my app, and it works fine.
Could you please add this enhancement to this npm as soon as possible?

**Reply by baronha** [reply]
I'm busy working on the upcoming version 2.0 so it might not be available to you soon

**Reply by baronha** [reply]
2.0: https://github.com/baronha/react-native-multiple-image-picker/discussions/182

**Comment by gkasireddy202** [comment]
Can you please provide the approximate ETA?

**Reply by baronha** [reply]
There is absolutely no specific time because maintaining this open source happens when I have time.

**Comment by gkasireddy202** [comment]
@baronha - Thanks for your update and support. I will be eagerly waiting for this enhancement.

**Reply by baronha** [reply]
Sure. When I have the results I will notify you again in this discussion

**Reply by gkasireddy202** [reply]
@baronha - Thanks.

**Comment by baronha** [comment]
I'm working on this branch. The results will be available to you soon
https://github.com/baronha/react-native-multiple-image-picker/pull/190

**Comment by gkasireddy202** [comment]
@baronha - Thanks for your update.

**Comment by baronha** [comment]
Result: 

https://github.com/user-attachments/assets/7afbc4cd-07a7-46b0-8501-9f1b3be70699

**Comment by baronha** [comment]
Done: https://github.com/baronha/react-native-multiple-image-picker/releases/tag/2.1.0

**Reply by gkasireddy202** [reply]
@baronha  - Thanks for your update. Does the latest version have to capture multiple images from the camera without dismissing the picker?

**Reply by baronha** [reply]
I only allow one photo at a time. And return results immediately, in addition you can also choose to crop after shooting

**Reply by gkasireddy202** [reply]
Is there any plan to implement features like capturing multiple images from the camera in the future without dismissing the picker?

**Reply by baronha** [reply]
Nah. I hadn't thought of that feature. I'm really sorry

**Reply by gkasireddy202** [reply]
@baronha - Thanks for your update. I will use this library.
